Formula/tfsec.rb

```diff
@@ -37,7 +37,7 @@ def install
 
     good_output = shell_output("#{bin}/tfsec #{testpath}/good")
     assert_match "No problems detected!", good_output
-    assert_no_match(/WARNING/, good_output)
+    refute_match(/WARNING/, good_output)
```

Comment:
This doesn't really require a regex, we can just do 
```suggestion
    refute_match("WARNING", good_output)
```